src/services/TrusteeRepository.ts

```diff
@@ -0,0 +1,30 @@
+import ITrusteeRepository from './ITrusteeRepository';
+import { TrusteeInput } from '@tpr/layout/lib/components/cards/trustee/context';
+import Data from '../data.json';
+
+class TrusteeRepository implements ITrusteeRepository {
+  WriteTrustee(trustee: TrusteeInput): void {
+    const toWrite = {
+      ...trustee,
+      id: this.getRandomInt(1000).toString(),
+      effectiveDate: new Date().toString(),
+      schemeRoleId: this.getRandomInt(1000).toString(),
+    };
+
+    Data.trustees.push(toWrite);
+  }
+  GetAllTrustees(): TrusteeInput[] {
+    const trustees = Data.trustees;
+    return trustees;
+  }
+
+  GetTrustee(id: string): TrusteeInput {
+    return Data.trustees[0];
```

Comment:
Did you mean to use the id as the array index here, rather than always getting the first trustee?

Reply:
Didn't mean to always return the first trustee, will fix now